.github/workflows/cloud_integration.yml

```diff
@@ -1,6 +1,9 @@
 name: Cloud integration
 on:
   push:
+    paths-ignore:
+    - '*.md'
```

Comment:
I think `*.md` is redundant since `**/*.md` searches all files as well as directories and subdirectories. This may be GH Actions specific though?

Reply:
Turns out, according to my tests, `**/*.md` doesn't catch files in the root dir so both conditions are required :man_shrugging:

Reply:
Interesting!